pkgs/development/tools/rome/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rome";
+  version = "11.0.0";
```

Comment:
12.0.0 got released upstream. Can you check if the ignored tests still fail with it?

Reply:
Yes, two tests still fail.